bin/build

```diff
@@ -7,4 +7,4 @@ echo -e "\n Formatting packages..."
   go fmt ./...
 
 echo -e "\nGenerating Binary..."
-go build -o $(dirname $0)/../out/slgo ./main/slgo.go
+go build -o $(dirname $0)/../out/slgo ../main/slgo.go
```

Comment:
I think this was my issue doing some development and testing in a Windows environment (we've been testing in Mac/Centos/Windows) and had to do with the GOPATH not being set right on my side. I'm asking internally though because I can't remember :/